lib/presentation/component/edit_workout_card.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
```

Comment:
Let's also move this file to the edit_workout package since it relies on the edit_workout_bloc